pom.xml

```diff
@@ -201,7 +201,7 @@
       <artifactId>lucene-benchmark</artifactId>
       <version>${LUCENE_VERSION}</version>
     </dependency>
-    <dependency> <!-- needed only for org.apache.lucene.search.CheckHits: candidate for cleanup -->
+    <dependency>
```

Comment:
Actually, as it turns out, we have test cases that also depend on this artifact.

Reply:
done!